.github/workflows/build-test.yml

```diff
@@ -618,17 +617,14 @@ jobs:
         uses: actions/setup-dotnet@v1.8.0
         with:
           dotnet-version: ${{env.DOTNET_VERSION}}
-        if: (matrix.test-suite == 'integration' || matrix.test-suite == 'notebook')
```

Comment:
So these run on all tests now?

Reply:
Oh I guess this was redundant.

Reply:
Oh crap, i remvoed the wrong thing, damn it

Reply:
Fixed